web/modules/custom/os2forms_queued_email/src/Plugin/WebformElement/Os2FormsQueuedEmailFileTrait.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Drupal\os2forms_queued_email\Plugin\WebformElement;
+
+use Drupal\os2forms_queued_email\Plugin\AdvancedQueue\JobType\QueuedEmail;
+use Drupal\webform\WebformSubmissionInterface;
+
+/**
+ * OS2Forms queued email file trait.
+ *
+ * Removes fileconent and adds flag static file flag.
```

Comment:
Do you work for the department of redundancy department?

```suggestion
 * Removes file content and adds static file flag.
```

Consider moving the comment to the `getEmailAttachments` implementation.

Reply:
Resolved!